app/components/EventDetails.js

```diff
@@ -45,11 +45,6 @@ const EventDetails = (props) => {
     }
   };
 
-
-  const formatDate = (dateString) => {
```

Comment:
Refer to https://github.com/STEM-E-Youth-Career-Development-Program/map-app/pull/3#issue-2141311340